content/client-lib-development-guide/features.textile

```diff
@@ -232,6 +232,17 @@ h3(#rest-auth). Auth
 ** @(RSA16b)@ If the library is provided with a token without the corresponding @TokenDetails@, then this holds a @TokenDetails@ instance in which only the @token@ attribute is populated with that token string
 ** @(RSA16c)@ Is set with the current token (if applicable) on instantiation and each time it is replaced, whether the result of an explicit @Auth#authorize@ operation, or a library-initiated renewal resulting from expiry or a token error response
 ** @(RSA16d)@ Is empty if there is no current token, including after a previous token has been determined to be invalid or expired, or if the library is using basic auth
+* @(RSA17)@ @Auth#revokeTokens@ function
+** @(RSA17a)@ Revokes tokens based on provided function arguments. It internally creates request for  "REST revokeTokens endpoint":https://ably.com/docs/api/rest-api#revoke-tokens and handles the response.
+** @(RSA17b)@ Method signature is @revokeTokens(Array<TokenRevocationTarget>. issuedBefore, allowReauthMargin)@.
+*** @(RSA17b1)@ @Array<TokenRevocationTarget>@ is an array of @TokenRevocationTarget@s to revoke. This array of token revocation targets must be used to construct the individual @TokenRevocationTarget@s in the form of "@TokenRevocationTarget#key@:@TokenRevocationTarget#value@" for the "targets" array in the request body.
+*** @(RSA17b2)@ @issuedBefore@ is an optional timestamp (in milliseconds since the epoch) to revoke tokens issued before this time.
+*** @(RSA17b3)@ @allowReauthMargin@ is an optional boolean that permits a token renewal cycle to take place without needing established connections to be dropped. Default for this value is false if not provided.
+** @(RSA17c)@ key to build the request must be retrieved from stored @Auth#AuthOptions#key@
+** @(RSA17d)@ If the @targets@ parameter is null or empty, then the function must throw an error with a @"targets cannot be null or empty"@ message.
+** @(RSA17e)@ If the @targets@ parameter is valid, then the function must create the request and execute it.
+** @(RSA17f)@ Function returns a @TokenRevocationResponse@ object that contains the result of the request if request is successful.
```

Comment:
"if request is successful" -> "if the request is successful"

Reply:
f9420343b5b5f7ea07c0254459b6c98886d7de19